webui/vendor/zendframework/zend-stdlib/src/ArrayObject.php

```diff
@@ -423,7 +423,7 @@ public function unserialize($data)
                     $this->setIteratorClass($v);
                     break;
                 case 'protectedProperties':
-                    continue;
+                    break;
```

Comment:
could you enlighten me how this is a functional change at all?
Didn't `continue` already move to the next step in `foreach()`?

Reply:
Nevermind... it isn't functional. But why does `continue` in the switch trigger a warning?

Reply:
PHP Warning:  "continue" targeting switch is equivalent to "break". Did you mean to use "continue 2"?

Reply:
- The break statement terminates the whole iteration of a loop whereas continue skips the current iteration.
- The break statement terminates the whole loop early whereas the continue brings the next iteration early.
- In a loop for switch, break acts as terminator for case only whereas continue 2 acts as terminator for case and skips the current iteration of loop.

Reply:
So break as referenced in the commit message was the actual fix done by upstream, we manually apply here.

Reply:
See upstream PR here: https://github.com/zendframework/zend-stdlib/pull/91